docs/api.md

```diff
@@ -31,8 +31,11 @@ curl --header "Authorization: Bearer ${KUBERNETES_USER_TOKEN}" \
      "https://${K8S_API_URL}/apis/token.kubevirt.io/v1alpha1/namespaces/${VMI_NAMESPACE}/virtualmachines/${VMI_NAME}/vnc?duration=${DURATION}"
 ```
 
-The `KUBERNETES_USER_TOKEN` variable is a bearer token used to authenticate with
-kubernetes API. It can be obtained using:
+In this example, we use a bearer token to authenticate the user with the Kubernetes API server.
+Kubernetes supports two main ways of authenticating, token and TLS client certificates, and other ways can be added
+using an authenticating proxy. More information can be found in the [documentation](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#authentication-strategies).
```

Comment:
```suggestion
Kubernetes supports two main ways of authenticating, token and TLS client certificates. Nevertheless, other ways can be added
using an authenticating proxy. More information can be found in the [documentation](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#authentication-strategies).
```

Reply:
That doesn't sound right to me. `Nevertheless` has a different meaning.

Reply:
humm maybe `Moreover`?

Reply:
I agree, separating this into two sentences makes it easier to understand.

Reply:
Doen